geohealthaccess/storage.py

```diff
@@ -40,57 +40,39 @@ def protocol(self):
     def path(self):
         """Return location path (without scheme/protocol)."""
         if self.protocol == "local":
-            return self.location
+            return self._raw_path
         else:
-            return self.location.split("://")[-1]
+            return self._raw_path.split("://")[-1]
+
+    def __str__(self):
+        return self._raw_path
 
 
 def get_s3fs():
     """Initialize a S3 filesystem from environment variables.
 
-    Uses the following environment variables:
-        * `S3_ACCESS_KEY`
-        * `S3_SECRET_KEY`
-        * `S3_REGION_NAME` (defaults to "us-east-1")
+    Automatically uses the following environment variables:
+        * `AWS_ACCESS_KEY_ID`
+        * `AWS_SECRET_ACCESS_KEY`
+        * `AWS_REGION` (defaults to "us-east-1")
         * `S3_ENDPOINT_URL` (defaults to "s3.amazonaws.com")
 
-    If `S3_SECRET_KEY`, anonymous access is used (public buckets only).
-
     Returns
     -------
     fs : S3FileSystem
         A FileSystem object from s3fs.
     """
     if not has_s3fs:
         raise ImportError("s3fs library is required when using s3 urls.")
-    anon = not bool(os.getenv("S3_SECRET_KEY"))
     return s3fs.S3FileSystem(
-        key=os.getenv("S3_ACCESS_KEY"),
-        secret=os.getenv("S3_SECRET_KEY"),
-        anon=anon,
-        client_kwargs={
-            "region_name": os.getenv("S3_REGION_NAME", "us-east-1"),
-            "endpoint_url": os.getenv("S3_ENDPOINT_URL"),
-        },
+        client_kwargs={"endpoint_url": os.getenv("S3_ENDPOINT_URL"),},
```

Comment:
Is the `AWS_REGION` automatically used by s3fs or do we need to add it in the `client_kwargs` ?